feature/folder/api/build.gradle.kts

```
@@ -0,0 +1,11 @@
+plugins {
+    id(ThunderbirdPlugins.Library.android)
+}
+
+android {
+    namespace = "net.thunderbird.feature.folder.api"
+}
+
+dependencies {
+    implementation(projects.core.mail.folder.api)
+}
```

This should be a pure Java module:

```suggestion
plugins {
    id(ThunderbirdPlugins.Library.jvm)
    alias(libs.plugins.android.lint)
}

dependencies {
    implementation(projects.core.mail.folder.api)
}
```